### PR TITLE
Hotfix 1.12.2 - Upgrade vaadin to 7.7.28 (#35)

### PR DESCRIPTION
**Pull request by KochTobi**
* Upgrade vaadin `7.7.17` -> `7.7.28`

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,20 @@ Changelog
...
+1.12.2 (2021-11-02)
+-------------------
+
+**Added**
+
+**Fixed**
+
+**Dependencies**
+
+* `vaadin:7.7.17` ->  `vaadin:7.7.28`
```
Comment (jenniferboedker) [inline CHANGELOG.rst:16]: * com.vaadin 7.7.17 -> 7.7.28 (addresses CVE-2021-37714)

**Review by jenniferboedker — CHANGES_REQUESTED** [review]
LGTM but you forgot to upgrad the version in the POM for the project to 1.12.2. After this is done I will approve

File: CHANGELOG.rst (hunk 1 above)
Comment (KochTobi) [reply CHANGELOG.rst:16]: good point. I will change it

**Review by jenniferboedker — APPROVED** [review]
🚀

**Comment by KochTobi** [comment]
1.12.2 (2021-11-02)
-------------------

**Added**

**Fixed**

**Dependencies**

* `com.vaadin:7.7.17` -> `7.7.28` (addresses CVE-2021-37714)

**Deprecated**

**Pull request merged**